packages/snap-tracker/src/types.ts

```diff
@@ -80,6 +80,10 @@ export interface BeaconContext {
 		type?: string;
 		id?: string;
 	};
+	controller?: {
+		type: string;
+		id: string;
+	};
```

Comment:
This should be removed, as it is only passed into the `TrackErrorEvent` payload.